Oliviier-dev/medRecord
# Implement Login and Signup Functionality

Description:
This PR adds the functionality for user authentication, including Signup and Login endpoints.

Signup: Allows users to create an account with their name, email, password, and role. Upon successful signup, a JWT token is generated and returned.
Login: Allows users to log in using their email and password. A JWT token is generated upon successful login for authenticated access.

Changes:
Added user signup and login routes.
Implemented JWT-based authentication.
How to Test:
Test the signup and login endpoints to ensure users can register, log in, and receive a valid JWT token for authentication.
